### PR TITLE
Cleanup globals

### PR DESCRIPTION
**Pull request by applmak**
Working towards fixing #2.

### CODE REVIEW

File: config/demo-playlist.json [hunk 1]
```diff
@@ -80,6 +80,25 @@
       "title": "Chicago Brick Live",
       "author": "Ian Atkinson"
     },
+    {
```
Comment (iatkinson) [inline config/demo-playlist.json]: Should this be in this PR?

File: client/modules/module.js [hunk 2]
```diff
@@ -90,16 +85,10 @@ define(function(require) {
...
         },
         require: fakeRequire.createEnvironment(exposedNodeModules),
-        ServerModuleInterface: moduleInterface.Server,
-        ClientModuleInterface: moduleInterface.Client,
-        Object: Object,
         Surface: Surface,
         CanvasSurface: CanvasSurface,
         P5Surface: P5Surface,
         ThreeJsSurface: ThreeJsSurface,
-        geometry: geometry,
```
Comment (iatkinson) [inline client/modules/module.js]: Now that geometry is not a global can you also remove it from the jshint whitelist?

File: config/demo-playlist.json (hunk 1 above)
Comment (applmak) [inline config/demo-playlist.json]: Nope! Rewrote history; fixed.

File: client/modules/module.js (hunk 2 above)
Comment (applmak) [inline client/modules/module.js]: Done, and did this for other such globals.

**Pull request merged**